packages/components/src/components/table/table-head-cell/table-head-cell-utils.ts

```diff
@@ -17,3 +18,7 @@ export const createSortedEventInitDictDetail = (sort: TableHeadCellSort): Custom
 export const isSortable = (active: boolean, direction: Direction): boolean => {
   return active !== undefined && direction !== undefined;
 };
+
+export const getTableTheme = (element: Element): Theme => {
```

Comment:
Can be removed as soon as sort button is reverted.

Reply:
yes, I'll change it

Reply:
Done